datasets/gov/vstt/saugojamos_rusys.csv

```diff
@@ -0,0 +1,33 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vstt/rusys,,,,,,,,,,,,,"Saugojamos rūšys,Saugojamų augalų, gyvūnų, grybų rūšių duomenys"
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,Rusis,,,,id,,4,,,Saugojamos rūšys,"Saugojamų augalų, gyvūnų, grybų rūšių duomenys"
+,,,,,id,string,,,,4,open,,Identifikavimo kodas,
+,,,,,stebejimas,date,D,,,4,open,,Stebėjimo data,
+,,,,,sukurimas,date,D,,,4,open,,Įrašo sukūrimo data,
+,,,,,komentaras,string,,,,2,open,,Radavietės būsenos pastabos,
+,,,,,skaicius,integer,U,,,4,open,,Stebėtos rūšies individų skaičius,
+,,,,,tankumas_ha,number,U/ha,,,4,open,,Stebėtos rūšies individų tankumas,
+,,,,,tankumas_kv,number,U/m^2,,,4,open,,Stebėtos rūšies individų tankumas,
+,,,,,rusiu_stebejimas,string,,,,2,open,,Rūšių Stebėjimas,Stebėtos rūšies individai: kita (tik augalams ir grybams)
+,,,,,pastabos,string,,,,2,open,,Stebėtojo pastabos,
+,,,,,anketos_busena,string,,,,2,open,,Anketos būsena iš klasifikatorius,
+,,,,,anketos_statusas,string,,,,4,open,,Anketos statusas,Anketos (stebėjimo) statusas iš klasifikatorius
+,,,,,,enum,,,"""Aktuali""",,,,,
+,,,,,,,,,"""Archyvinė""",,,,,
+,,,,,biotopas,string,,,,4,open,,Biotopas,Stebėjimo biotopas (buveinė) ar kita informacija
+,,,,,gyvunu_stebejimas,string,,,,4,open,,Veiklos požymis,Gyvūno veiklos požymiai iš klasifikatoriaus
+,,,,,evoliucija,string,,,,4,open,,Vystymosi stadija,Gyvūno ar augalo vystymosi stadija iš klasifikatoriaus
+,,,,,radavietes_busena,string,,,,4,open,,Radavietės būsena,
+,,,,,rusis_lot,string,,,,4,open,,Rūšies pavadinimas lotyniškai,
+,,,,,rusis_lt,string,,,,4,open,,Rūšies pavadinimas lietuviškai,
+,,,,,klase,string,,,,4,open,,Klasės,
+,,,,,tipas,string,,,,4,open,,Tipas,
+,,,,,karalyste,string,,,,4,open,,Karalystė,
+,,,,,cell_id,integer,,,,4,open,,Gardelės identifikavimo numeris,
+,,,,,sluoksnis,string,,,,4,open,,Sluoksnio geometrijos tipas,
+,,,,,,enum,,,"""taskas""",,,,Taškas,"Stebėjimo taškas (x, y koordinatės). Žymimas kada stebimas objektas yra nedidelio ploto, kada fiziškai ploto neįmanoma įbraižyti, pvz. vienas augalas arba keras, vienas individas ar susitelkusi individų grupė. Paprastai tai objektai, kurie neviršija 5-10 m skersmens,"
+,,,,,,,,,"""poligonas""",,,,Poligonas,"Stebėjimo plotas (plotas hektarais, kvadratiniais metrais ir kt.). Žymimas kada objektas tolygiai išsidėstęs erdvėje ir įmanoma įbraižyti jo plotą. Dažniausiai tai didesni nei 5-10 m skersmens plotai, kuriuose įvairiu tankumu pasiskirstę stebimi objektai, pvz. jei yra pieva su retu augalu, kurioje tas augalas randamas beveik visur, gana tolygiai pasiskirstęs. Tokiu atveju braižomas plotas visai pievai."
+,,,,,,,,,"""linija""",,,,Linija,"Stebėjimo linija (ilgis metrais). Žymima kada stebimas objektas yra nedidelio pločio, tačiau išsidriekęs erdvėje į ilgį, pvz. miško pakraštyje ar palei keliuką, išsidriekusi augalų eilė, kurios plotis neviršija 5-10 m,"
+,,,,,geometrija,geometry(3346),m,,,3,open,,Ploto geometrija,
```

Comment:
```suggestion
,,,,,geometrija,geometry(3346),m,,,4,open,,Ploto geometrija,
```

Reply:
sluoksnio stulpelis nusako, kaip randama ir saugojama rūšis, todėl ji negali būti rišama prie stulpelio geometrija.